src/FSharpLint.Console/Program.fs

```diff
@@ -22,9 +22,10 @@ type private FileType =
 
 type private ExitCode = 
     | Error = -1
-    | Success = 0
+    | Success = 0 // for `fix` when the file is changed and for `fix --check` when there are no fixes available. 
     | NoSuchRuleName = 1
-    | NoSuggestedFix = 2
+    | NoSuggestedFix = 2 // only for fix (without --check)
+    | FixExists = 3 //only for fix --check
```

Comment:
@parhamsaremi nit: let's always put comments on top of code (not next to it), to avoid next devs to have to horizontal-scroll the document

Reply:
> > Co-authored-by: parhamsaremi [parhaamsaremi@gmail.com](mailto:parhaamsaremi@gmail.com)
> 
> Nit, the bit before the e-mail address should be in Name Surname format, not username. Unless you have anything against that?

hmmmm OK sure :thinking:  I don't have anything against it, I just preferred my username, but since my username is my Name and Surname I don't see any problem switching.

Reply:
ok sure